bandit/blacklists/imports.py

```diff
@@ -308,7 +311,13 @@ def gen_blacklist():
             "import_xml_etree",
             "B405",
             issue.Cwe.IMPROPER_INPUT_VALIDATION,
-            ["xml.etree.cElementTree", "xml.etree.ElementTree"],
+            [
+                "xml.etree.cElementTree",
+                "xml.etree.ElementTree.XMLParser",
+                "xml.etree.ElementTree.fromstring",
+                "xml.etree.ElementTree.iterparse",
+                "xml.etree.ElementTree.parse",
```

Comment:
I don't believe you can import `fromstring`, `iterparse`, or `parse` as a result this doesn't do what you want.

Reply:
Thanks for your response. I think you misunderstand with `ElementTree`, which in this case is a module name, i don't know why the stdlib named the [ElementTree](https://github.com/python/cpython/blob/main/Lib/xml/etree/ElementTree.py) module like the class name. The fact this code runs well.

```python
>>> from xml.etree.ElementTree import parse, iterparse, fromstring, XMLParser
>>> parse
<function parse at 0x00000206D1BE44A0>
>>> iterparse
<function iterparse at 0x00000206D1BE4540>
>>> fromstring
<function XML at 0x00000206D1BE45E0>
>>> XMLParser
<class 'xml.etree.ElementTree.XMLParser'>
>>>
```

And this more clear

```python
>>> from xml.etree import ElementTree
>>> ElementTree
<module 'xml.etree.ElementTree' from 'C:\\Users\\User\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\xml\\etree\\ElementTree.py'>
>>> ElementTree.ElementTree
<class 'xml.etree.ElementTree.ElementTree'>
>>>
```

Reply:
Ah sorry, looks like B314 rules already cover it. dumb me.

Reply:
What do you think, because rules B405 are too general and make the entire contents of the `ElementTree` module seem dangerous from a security perspective even though there are functions or classes that are not dangerous, for example `Element` and that is needed for typing. Do I need to add all the functions or classes that are dangerous and not include those that have nothing to do with security as I have listed above.

This code will warn B405 rules. But after changes in this PR, this code will not warn B405.

```python
import xml.etree.ElementTree
```

And in order to raise B405 this code will although bandit only warn the `fromstring` and the rest is not.

```python
from xml.etree.ElementTree import fromstring, parse, iterparse, XMLParser
```



This code is result in `ModuleNotFoundError`.

```python
import xml.etree.ElementTree.fromstring
import xml.etree.ElementTree.parse
import xml.etree.ElementTree.iterparse
import xml.etree.ElementTree.XMLParser
```